### PR TITLE
add php8.3 version

### PR DESCRIPTION
**Pull request by juliangut**
Empty upcoming list until php8.4 rc comes

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -33,7 +34,6 @@ let upcomingVersion = '';
 
 if (process.env.INPUT_UPCOMINGRELEASES === 'true') {
     [
-        '8.3'
     ].forEach(function (version) {
```
Comment (WyriHaximus) [inline main.js:37]: Can you change this into 8.4 and command the line out?

**Review by WyriHaximus — CHANGES_REQUESTED** [review]
Thanks for looking into this, can you update the tests as well?

**Comment by WyriHaximus** [comment]
> I've reviewd https://github.com/shivammathur/setup-php#tada-php-support indeed has 8.4 as an option

That poses an interesting challenge. Personally I only use that option when the first alpha is out and I've started building it through https://github.com/wyrihaximusnet/docker-php and then update the upcoming release here. So going to `8.4` for the upcoming release isn't an option. However, adding it as a nightly toggle is the way to forward IMHO.

**Comment by juliangut** [comment]
I'd tend to agree with you, but as the README file states "The 4rth output is 
`upcoming` and will be populated with the upcoming but unreleased next minor or major version of PHP" as long as that version is available in shivammathur/setup-php and thus can be used it is the next unreleased version

We can simply leave the upcoming list empty until first alpha is available or event maybe RC, but we can argue as well it is already in a separate configuration sot it is end user's responsibility for configuring it

If you want I can empty the upcoming releases array again, and you can implement a new nightly flag

**Comment by juliangut** [comment]
I've just made the change, hope the tests pass

Of course you can tag me

**Comment by WyriHaximus** [comment]
Will tag you in that nightly, and got an extension detection upgrade to do as well before releasing the next version

**Pull request merged**